docs/troubleshooting.asciidoc

```diff
@@ -28,3 +28,59 @@ include::copied-from-beats/getting-help.asciidoc[]
 == Debug
 
 include::copied-from-beats/debugging.asciidoc[]
+
+== Common problems
+=== 503: Queue is full
+APM Server has an internal queue which stores documents until they can be delivered to Elasticsearch.
+It is designed this way to alleviate problems that might occur if Elasticsearch is intermittently unavailable or if a large spike of data arrives at the APM Server at the same time.
+
+When the internal queue is full, APM Server will return a HTTP 503 status with the message "Queue is full".
```

Comment:
Add a line break after punctuations, i.e:

```diff
-When the internal queue is full, APM Server will return a HTTP 503 status with the message "Queue is full".
+When the internal queue is full,
+APM Server will return a HTTP 503 status with the message "Queue is full".
```

The same goes for the rest of the document, but I'll comment more on it